display_list/display_list_builder.h

```diff
@@ -472,7 +474,7 @@ class DisplayListBuilder final : public virtual Dispatcher,
   std::shared_ptr<const DlColorSource> current_color_source_;
   std::shared_ptr<const DlColorFilter> current_color_filter_;
   std::shared_ptr<DlImageFilter> current_image_filter_;
-  sk_sp<SkPathEffect> current_path_effect_;
+  std::shared_ptr<const DlPathEffect> current_path_effect_;
```

Comment:
This is odd, these were all recently merged into DlPaint (except for PathEffect which was not yet Dl-ified). That happened before the Impeller cut over so I'm not sure why we are seeing this code in this state...?

Reply:
Ignore this comment. The DlPaint PR has landed, but the PR that moves all of these fields into a DlPaint object has not landed yet.

Reply:
done